components/esp32_ble_presense/esp32_ble_presense.cpp

```diff
@@ -75,7 +85,8 @@ void ESP32_BLE_Presense::setup() {
     pBLEScan->setWindow(bleScanWindow);
     pBLEScan->setActiveScan(false);
     pBLEScan->setMaxResults(0);
-    configTime(0, 0, "pool.ntp.org");
+    if (!rtc)
+        configTime(0, 0, "pool.ntp.org");
```

Comment:
here we fallback to the old code - I'm not sure this was necessary, and I think using homeassistant instead of ntp is probably more efficient/consistent.